src/patterns/data/default/entity_attribute.tsv

```diff
@@ -7242,3 +7242,12 @@ OBA:1001089	multicellular organism sex	UBERON:0000468	multicellular organism	PAT
 OBA:1001090	feather color	UBERON:0000022	feather	PATO:0000014	color
 OBA:1001091	egg shell trait	UBERON:0005079	eggshell	PATO:0000001	quality
 OBA:1001092	egg shell color	UBERON:0005079	eggshell	PATO:0000014	color
+OBA:2040143	metafemur length	HAO:0001140	metafemur	PATO:0000122	length
```

Comment:
Remove, make uberon issue requesting metafemur, make uberon PR adding metafemur, xrefing to HAO.

Introduce policy: NO species specific anatomy ontology references in OBA. Only UBeron.